Article/Article.js

```diff
@@ -3,18 +3,23 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
+    console.log(this.expandButton)
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerHTML = 'Expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
```

Comment:
`.textContent` would also be apt for updating text. `.innerHTML` is actually intended for changing HTML markup (which can be a cybersecurity risk).